README.md

```diff
@@ -1,5 +1,30 @@
 # repo-policy-compliance
-Tool to check a GitHub repository for compliance with policy
+A Juju charm of a Flask application to check if a GitHub repository aligns
```

Comment:
I am missing a bit the rationale, maybe add the following line : 
"The charm is intended to be used in conjunction with the [Github runner](https://charmhub.io/github-runner) charm to prevent malicious workflow execution in internal infrastructure."

Reply:
Thanks for the suggestion, added!